skrifa/src/variation.rs

```diff
@@ -168,9 +168,8 @@ impl<'a> AxisCollection<'a> {
     /// * Omitted settings are set to 0.0, representing the default position
     ///     in variation space.
     ///
-    /// # Panics
-    ///
-    /// Panics if `location.len() < self.len()`.
+    /// If `location.len() < self.len()`, the omitted coordinates will assume
+    /// their default positions.
```

Comment:
Isn't this comment duplicative of the "Omitted settings are set to ..." part?

Reply:
Hm. Also it's wrong because location.len could be >= self.len while still not specifying positions on all axes so ... just delete and rely on the Ommitted bit?